snapcraft.yaml

```diff
@@ -1,8 +1,7 @@
 name: owncloud
 version: 9.0.0ubuntu1
 summary: ownCloud
-description: ownCloud running on Apache with MySQL. This is currently in beta.
-icon: icon.svg
```

Comment:
The bug that you were hitting has been released in the most recent release of Snapcraft. Please update when you're able and put this back.